components/core/tabs/Tabs.vue

```diff
@@ -48,7 +48,7 @@ export default {
 }
 .tabs__headers > .header {
     @apply inline-block ml-1 py-2 px-2;
-    @apply text-sm text-center font-black;
+    @apply text-base text-center font-black;
```

Comment:
```suggestion
    @apply text-sm md:text-base text-center font-black;
```

Use a smaller font size in case the headers of tabs are displayed in two lines on iPhone 5/SE.
![image](https://user-images.githubusercontent.com/24987826/129467081-2cb2f16a-9b10-474a-95b9-a270c495f950.png)